docs/desktop-wpf/fundamentals/styles-and-templates-overview.md

```diff
@@ -1,11 +1,190 @@
 ---
```

Comment:
file names should avoid words like and

https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=master#name-your-files

Reply:
I'll follow up with another PR to remove small words out of all of the files.